src/components/Category/Category.jsx

```diff
@@ -1,26 +1,33 @@
-/* eslint-disable no-unused-vars */
-import React from 'react';
-import ReactDOM from 'react-dom';
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] Just like the previous review said kindly solve linters errors intead of disabling them, note that they make your code more maintenable and prevent future bags

Reply:
Hi @andersonlebon 
Thank you very much for your feedback. Kindly note that this comment is only for disabling the next line no-unused-vars as I will receive an error and the app will not work if it's not there even though I will not use it inside the file, but it's also giving me a linter error. This is why I had to disable eslint for this case and this line only. Do you know of any way I can use react to fix this error without disabling eslint no-unused-vars?